internal/runbits/requirements/requirements.go

```diff
@@ -558,99 +555,6 @@ func (r *RequirementOperation) resolveRequirement(requirement *Requirement) erro
 	return nil
 }
 
-func (r *RequirementOperation) cveReport(artifactChangeset buildplan.ArtifactChangeset, requirements ...*Requirement) error {
-	if r.shouldSkipCVEs(requirements...) {
-		logging.Debug("Skipping CVE reporting")
-		return nil
-	}
-
-	names := requirementNames(requirements...)
-	pg := output.StartSpinner(r.Output, locale.Tr("progress_cve_search", strings.Join(names, ", ")), constants.TerminalAnimationInterval)
-
-	var ingredients []*request.Ingredient
-	for _, requirement := range requirements {
-		if requirement.Operation == types.OperationRemoved {
-			continue
-		}
```

Comment:
This is the only functional change in the PR. I removed the `requirements` argument and construct the list of names from the `artifactChangeset`. I also removed the top-level iteration over `requirements`, which is superfluous.